src/framework/cmake/MuseDeclareOptions.cmake

```
@@ -19,7 +19,7 @@ option(MUSE_MODULE_AUDIO_EXPORT "Enable audio export" ON)
 # 1 - worker
 # 2 - driver callback
 # 3 - worker - RPC, driver callback - process
-set(MUSE_MODULE_AUDIO_WORKMODE 1 CACHE INT "Audio subsystem work mode")
+set(MUSE_MODULE_AUDIO_WORKMODE CACHE INT "Audio subsystem work mode" 1)
```

The problem is that there is no `INT` type in CMake. `STRING` must be used instead.
(https://cmake.org/cmake/help/v3.31/command/set.html#set-cache-entry)
(don't know how this affects it's use in `src\framework\cmake\muse_framework_config.h.in`

```suggestion
set(MUSE_MODULE_AUDIO_WORKMODE 1 CACHE STRING "Audio subsystem work mode")
```